x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -568,6 +569,11 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
     confirmManualRuleRun,
   } = useManualRuleRunConfirmation();
 
+  const groupTakeActionItems = useGroupTakeActionsItems({
```

Comment:
An interesting fact is that `groupTakeActionItems` is a render function returning items. Name might be confusing.

Reply:
Agreed, but like mentioned above, I kept this naming because I wanted to stay consistent with the rest of the props in the component and in the `kbn-grouping` package